nixnet_examples/programmatic_database_api.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import pprint
+import six
+import time
+
+import nixnet
+from nixnet import constants
+from nixnet import types
+from nixnet.system import system
+
+
+pp = pprint.PrettyPrinter(indent=4)
+
+
+def write_can_frame_using_alias(database_name):
+    cluster_name = 'CAN_Cluster'
+    output_frame = 'CANEventFrame1'
+    interface = 'CAN1'
+
+    with nixnet.FrameOutQueuedSession(
+            interface,
+            database_name,
+            cluster_name,
+            output_frame) as output_session:
+        terminated_cable = six.moves.input('Are you using a terminated cable (Y or N)? ')
+        if terminated_cable.lower() == "y":
+            output_session.intf.can_term = constants.CanTerm.OFF
+        elif terminated_cable.lower() == "n":
+            output_session.intf.can_term = constants.CanTerm.ON
+        else:
+            print("Unrecognised input ({}), assuming 'n'".format(terminated_cable))
+            output_session.intf.can_term = constants.CanTerm.ON
+
+        user_value = six.moves.input('Enter payload [int, int]: ')
+        try:
+            payload_list = [int(x.strip()) for x in user_value.split(",")]
+        except ValueError:
+            payload_list = [2, 4, 8, 16]
+            print('Unrecognized input ({}). Setting data buffer to {}', user_value, payload_list)
+
+        id = types.CanIdentifier(0)
+        payload = bytearray(payload_list)
+        frame = types.CanFrame(id, constants.FrameType.CAN_DATA, payload)
+
+        print('Writing CAN frames using ({}) alias:', database_name)
+
+        i = 0
+        while i < 4:
+            for index, byte in enumerate(payload):
+                payload[index] = byte + i
+
+            frame.payload = payload
+            output_session.frames.write([frame])
+            print('Sent frame with ID %s payload: %s' % (id, payload))
+
+def main():
```

Comment:
I'm assuming this is more so to showcase the API work you are doing rather than an example because I wouldn't expect an example to be creating multiple aliases.